src/view/ConversationWindow.vala

```diff
@@ -288,10 +292,15 @@ namespace Venom {
     }
 
     public signal void send();
-
+    public signal void insert_new_line();
+    
     public bool on_key_press_event(Gdk.EventKey event) {
-      if (event.keyval == key.accel_key && event.state == key.accel_mods) {
-        send();
+      if (event.keyval in new uint[]{Gdk.Key.Return, Gdk.Key.KP_Enter}) {
```

Comment:
this will discard any user set preferences on which key to press for sending

Reply:
Mh, yeah. Will fix.